greeting-service/src/main/resources/infinispan-conf.xml

```diff
@@ -4,9 +4,6 @@
         xmlns="urn:infinispan:config:9.1">
 
     <cache-container name="container" default-cache="default">
-        <local-cache name="default">
-            <expiration lifespan="5000"/>
-        </local-cache>
```

Comment:
You probably should have removed just the `expiration` line, leaving the `local-cache` intact.

Reply:
You're right, all do that

Reply:
Done